tinydrive/tinydrive-api/standalone.md

```diff
@@ -21,7 +21,7 @@ In order to use {{site.cloudfilemanager}} in standalone mode you will need to ad
 <script>
 tinydrive.pick({
   token_provider: '/your-local/jwt-provider'
-}).then((result) => {
+}).then( function (result) {
```

Comment:
There probably shouldn't be a space before `function` here.

Reply:
Fixed in https://github.com/tinymce/tinymce-docs/pull/1645/commits/50d665fdbcdb9afd3395dccf0ff6eba409f4895d